framework/resource/metricsresource/metric.go

```diff
@@ -0,0 +1,46 @@
+package metricsresource
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	PrometheusNamespace = "operatorkit"
+	PrometheusSubsystem = "framework"
+)
+
+var (
+	operationCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: PrometheusNamespace,
+			Subsystem: PrometheusSubsystem,
```

Comment:
Isn't it a good idea to use `framework.PrometheusNamespace` instead?

Reply:
Uncertain. I am ok with the current state. Let me know when you have strong feelings for this.